### PR TITLE
adding support for uploading csv files

### PR DESCRIPTION
**Pull request by PalSzabolcs**
https://lumoslabs.atlassian.net/browse/DE-12365
@sgrgic @dtboctor this change is required in order for us to upload CSV files via the API

### CODE REVIEW

File: lib/leanplum_api/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module LeanplumApi
-  VERSION = '4.0.3'
```
Comment (slpsys) [inline lib/leanplum_api/version.rb]: This is actually a minor bump (not a patch), given that it's new, backward-compatible feature, per [semver](https://semver.org/).

Comment (PalSzabolcs) [reply lib/leanplum_api/version.rb]: ok, sure, in contrast to the prev change that was a bug fix this is adding a new feature, so yeah, will change!

Comment (PalSzabolcs) [reply lib/leanplum_api/version.rb]: @slpsys it's changed now

Comment (slpsys) [reply lib/leanplum_api/version.rb]: 👍

**Pull request merged**